aws/microservices/tb-cache-configmap.yml

```diff
@@ -22,8 +22,8 @@ metadata:
   labels:
     name: tb-cache-config
 data:
-  CACHE_TYPE: redis
```

Comment:
Please check availiable options for [CACHE_TYPE](https://thingsboard.io/docs/user-guide/install/config/#cache-settings-parameters). There is no such option as valkey.